.travis.yml

```diff
@@ -24,6 +24,7 @@ before_install:
     - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-metadata.git
     - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-test.git
     - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-variation.git
+    - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-compara.git
```

Comment:
This file has actually changed significantly on master since you've opened your pull-request, see https://github.com/Ensembl/ensembl-datacheck/blob/master/.travis.yml#L20-L32
That's why GitHub says there is a conflict on this file. It might be best to rebase the branch ? That would also address the other conflict